lib/yang.h

```diff
@@ -511,6 +511,9 @@ extern void yang_init(bool embedded_modules);
  */
 extern void yang_terminate(void);
 
+extern const struct lyd_node *yang_dnode_get_pleaf(const struct lyd_node *dnode,
+						   const char *lname);
```

Comment:
spacing looks off have you properly run clang-format?

Reply:
done